### PR TITLE
Update the supported environments topic for 0.40.0 release

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
Updated the supported environments topic for 0.40.0 release.

Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

File: docs/openj9_support.md [hunk 1]
```diff
@@ -42,14 +42,13 @@ columns will be removed over time.
...
+| 0.40.0         | Jul 2023   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |              |
+| 0.40.1         | Sep 2023 (1, 3)   |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |   :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>          |   :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |
```
Comment (pshipton) [inline docs/openj9_support.md:51]: There is no 0.40.1 release in Sept. You can add a 0.41.0 release in Oct, which may or may not include support for JDK21.

File: docs/openj9_support.md [hunk 2]
```diff
@@ -42,14 +42,13 @@ columns will be removed over time.
...
-| 0.40.0         | Jul 2023 (1)   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |
+| OpenJ9 release | Release date  | JDK8 (LTS) | JDK11 (LTS) | JDK17 (LTS) | JDK19 | JDK20 |  JDK21 (LTS)  |
+|----------------|---------------|------------|-------------|-------------|-------|-------|---------|
+| 0.37.0         | Mar 2023 (2)  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |            |          |
```
Comment (pshipton) [inline docs/openj9_support.md:47]: Can we remove 0.37 from the table and remove JDK19?

Comment (Sreekala-Gopakumar) [reply docs/openj9_support.md:47]: Ok. Any other changes to be done in this Supported environments topic for Release 0.40.0. Thanks!

https://eclipse-openj9.github.io/openj9-docs-staging/1126/openj9_support/

Comment (pshipton) [reply docs/openj9_support.md:47]: 0.40.0 looks good to me.

File: docs/openj9_support.md (hunk 1 above)
Comment (Sreekala-Gopakumar) [reply docs/openj9_support.md:51]: Will this 0.41.0 release be one of those releases that support a new LTS OpenJDK release only or all the listed JDKs including 21?

File: docs/openj9_support.md (hunk 2 above)
Comment (Sreekala-Gopakumar) [reply docs/openj9_support.md:47]: So no changes in the the minimum operating system levels or operating system and compiler levels for the build systems?

File: docs/openj9_support.md (hunk 1 above)
Comment (pshipton) [reply docs/openj9_support.md:51]: Removing JDK21 for now is fine with me.

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - Please check and confirm. Thanks!

File: docs/openj9_support.md [hunk 3]
```diff
@@ -42,14 +42,12 @@ columns will be removed over time.
...
+| 0.39.0         | Jun 2023 (2)  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>      |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |
+| 0.40.0         | Jul 2023   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |
+| 0.41.0         | Oct 2023 (1)   |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |
```
Comment (pshipton) [inline docs/openj9_support.md:50]: JDK20 is not included with 0.41.0 as it will be out of support.

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

File: docs/openj9_support.md (hunk 3 above)
Comment (Sreekala-Gopakumar) [reply docs/openj9_support.md]: Updated the document. Please merge if there are no other changes. Thanks!

**Review by pshipton — APPROVED** [review]

**Pull request merged**